client/src/app/metadata-schema-form/metadata-form/metadata-form.component.html

```diff
@@ -1,10 +1,10 @@
 <form *ngIf="done" class="vf-form" [formGroup]="metadataForm.formGroup" (submit)="onSubmit($event)">
 
-  <mat-tab-group *ngIf="config.layout.tabs.length > 1"
+  <mat-tab-group *ngIf="visibleTabs(config.layout.tabs) > 1"
```

Comment:
You could have 1 function to return a list of tabs to be displayed (visible) and remove the `*ngIf=showTab`.  I believe that would also improve the rendering of the template.